packages/leancode_lint/lib/analysis_options.yaml

```diff
@@ -3,6 +3,203 @@ analyzer:
+    prefer_void_to_null: warning
+    provide_deprecation_message: warning
+    recursive_getters: warning
+    require_trailing_commas: warning
+    secure_pubspec_urls: warning
+    sized_box_for_whitespace: warning
+    sized_box_shrink_expand: warning
+    slash_for_doc_comments: warning
+    sort_child_properties_last: warning
+    sort_constructors_first: warning
+    sort_pub_dependencies: warning
+    sort_unnamed_constructors_first: warning
+    test_types_in_equals: warning
+    throw_in_finally: warning
+    tighten_type_of_initializing_formals: warning
+    type_annotate_public_apis: warning
+    type_init_formals: warning
+    type_literal_in_constant_pattern: warning
+    unawaited_futures: warning
+    unnecessary_await_in_return: warning
+    unnecessary_brace_in_string_interps: warning
+    unnecessary_breaks: warning
+    unnecessary_const: warning
+    unnecessary_constructor_name: warning
+    unnecessary_getters_setters: warning
+    unnecessary_lambdas: warning
+    unnecessary_late: warning
+    unnecessary_library_directive: warning
+    unnecessary_new: warning
+    unnecessary_null_aware_assignments: warning
+    unnecessary_null_aware_operator_on_extension_on_nullable: warning
+    unnecessary_null_checks: warning
+    unnecessary_null_in_if_null_operators: warning
+    unnecessary_nullable_for_final_variable_declarations: warning
+    unnecessary_overrides: warning
+    unnecessary_parenthesis: warning
+    unnecessary_raw_strings: warning
+    unnecessary_statements: warning
+    unnecessary_string_escapes: warning
+    unnecessary_string_interpolations: warning
+    unnecessary_this: warning
+    unnecessary_to_list_in_spreads: warning
+    unrelated_type_equality_checks: warning
+    unsafe_html: warning
+    use_enums: warning
+    use_full_hex_values_for_flutter_colors: warning
+    use_function_type_syntax_for_parameters: warning
+    use_if_null_to_convert_nulls_to_bools: warning
+    use_is_even_rather_than_modulo: warning
+    use_key_in_widget_constructors: warning
+    use_named_constants: warning
+    use_raw_strings: warning
+    use_rethrow_when_possible: warning
+    use_string_buffers: warning
+    use_string_in_part_of_directives: warning
+    use_super_parameters: warning
+    use_test_throws_matchers: warning
+    use_to_and_as_if_applicable: warning
+    valid_regexps: warning
+    void_checks: warning
+    # todo disagnostic messages
+    fixme: warning
+    hack: ignore
+    todo: info
+    undone: info
+    # hint diagnostic mesages
+    can_be_null_after_null_aware: warning
+    deprecated_colon_for_default_value: warning
+    deprecated_export_use: info
+    deprecated_member_use: info
+    division_optimization: warning
+    import_deferred_library_with_load_function: warning
+    import_of_legacy_library_into_null_safe: warning
+    unignorable_ignore: warning
+    unnecessary_cast: warning
+    unnecessary_import: warning
+    unreachable_switch_case: warning
+    unused_element: warning
```

Comment:
hint?

Reply:
Yep the ones from this file https://github.com/dart-lang/sdk/blob/main/pkg/analyzer/lib/src/dart/error/hint_codes.g.dart